### ISSUE TITLE
Classifier returns logits not soft probablities

### ISSUE THREAD

**Issue by shubham303** [antoniobarbalau/black-box-ripper#3]
At this line https://github.com/antoniobarbalau/black-box-ripper/blob/f461e140979bf43200bd4ec6872731f07a2ca15f/torch_optimizer.py#L106 we are getting logits.

Which means c value can be outside the range [0,1]. I think the condition c<0.9 needs to be changed.
